components/reusable-components/reusable-table/ReusableTable.tsx

```diff
@@ -70,7 +49,7 @@ const ReusableTable = <T extends { id: string }>({
     return data;
   }, [data, sortState]);
 
-  if (data?.length === 0) {
+  if (data.length === 0) {
```

Comment:
return the optional chaining here. it is a good practice to use it in case the data we get is undefined or null, we won't get errors.